docker-hub/orgs.md

```diff
@@ -2,27 +2,22 @@
 description: Docker Hub Teams and Organizations
 keywords:
 - Docker, docker, registry, teams, organizations, plans, Dockerfile, Docker Hub, docs,  documentation
-menu:
-  main:
-    parent: mn_pubhub
-    weight: -80
-title: Teams & Organizations
+title: Organizations and teams
```

Comment:
Since we're specifying "in Docker Cloud" we might as well be specific on this one too and add "Docker Hub".